compliance/collection/main.go

```diff
@@ -156,7 +159,7 @@ func manageSendToSensor(ctx context.Context, cli sensor.ComplianceService_Commun
 	}
 }
 
-func manageNodeScanLoop(ctx context.Context, i intervals.NodeScanIntervals, scanner nodeinventorizer.NodeInventorizer) <-chan *sensor.MsgFromCompliance {
+func manageNodeScanLoop(ctx context.Context, i intervals.NodeScanIntervals, scanner scannerV1.NodeInventoryServiceClient) <-chan *sensor.MsgFromCompliance {
```

Comment:
If `scanner scannerV1.NodeInventoryServiceClient` is an interface, then all is good. If it is an implementation, I would pass it as a pointer `scanner *scannerV1.NodeInventoryServiceClient`